indico/modules/users/templates/dashboard.html

```diff
@@ -82,6 +82,36 @@ <h3>{{ user.full_name }}</h3>
                         </a>
                     </div>
                 </div>
+                <div id="happeningCategories" class="dashboard-box">
+                    <h3>{% trans %}Your unlisted events{% endtrans %}</h3>
+                    <ol>
+                        {% if not unlisted_events %}
+                            <li class="no-event">
+                                <span class="event-title italic text-superfluous">
+                                    {% trans %}You have no unlisted events.{% endtrans %}
+                                </span>
+                            </li>
```

Comment:
IIRC Jinja lets you put an `else` block in a `for` loop that executes when there was nothing to iterate over - so you could use this for the no-event case.

Reply:
But do we want to show this box at all if there aren't unlisted events? Especially in case someone is not allowed to create them I would not want this to show up.